### PR TITLE
Corrected broken link for Certificates

### PR DESCRIPTION
**Pull request by cadasi**
We have a link to download a certificate chain that customers need in their Edge Server, but that is broken and pointing to an internal Microsoft Server.
I fixed that pointing to an article with certificate chains that will do the same job.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Lync/LyncServer/lync-server-2013-migrating-lync-online-users-to-lync-on-premises.md [hunk 1]
```diff
@@ -60,36 +60,8 @@ _**Topic Last Modified:** 2015-11-13_
             New-CsHostingProvider -Identity LyncOnline -ProxyFqdn "sipfed.online.lync.com" -Enabled $true -EnabledSharedAddressSpace $true -HostsOCSUsers $true -VerificationLevel UseSourceVerification -IsLocal $false -AutodiscoverUrl https://webdir.online.lync.com/Autodiscover/AutodiscoverService.svc/root
            ```
 
-2.  Confirm that on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online, as shown in the following table. You can download this chain here: [https://corp.sts.microsoft.com/Onboard/ADFS\_Onboarding\_Pack/corp\_sts\_certs.zip](https://corp.sts.microsoft.com/onboard/adfs_onboarding_pack/corp_sts_certs.zip) .
+2.  Confirm that on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/en-ie/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
```
Comment (JohanFreelancer9) [inline Lync/LyncServer/lync-server-2013-migrating-lync-online-users-to-lync-on-premises.md]:
```suggestion
2.  Confirm that, on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/en-ie/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
```

File: Lync/LyncServer/lync-server-2013-migrating-lync-online-users-to-lync-on-premises.md [hunk 2]
```diff
@@ -60,36 +60,8 @@ _**Topic Last Modified:** 2015-11-13_
             New-CsHostingProvider -Identity LyncOnline -ProxyFqdn "sipfed.online.lync.com" -Enabled $true -EnabledSharedAddressSpace $true -HostsOCSUsers $true -VerificationLevel UseSourceVerification -IsLocal $false -AutodiscoverUrl https://webdir.online.lync.com/Autodiscover/AutodiscoverService.svc/root
            ```
 
-2.  Confirm that on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online, as shown in the following table. You can download this chain here: [https://corp.sts.microsoft.com/Onboard/ADFS\_Onboarding\_Pack/corp\_sts\_certs.zip](https://corp.sts.microsoft.com/onboard/adfs_onboarding_pack/corp_sts_certs.zip) .
+2.  Confirm that, on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/en-ie/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
```
Comment (nenonix) [inline Lync/LyncServer/lync-server-2013-migrating-lync-online-users-to-lync-on-premises.md]:
```suggestion
2.  Confirm that, on your on-premises Edge Servers, you have the certificate chain that enables connection to Lync Online. You can download this chain here: https://support.office.com/article/office-365-certificate-chains-0c03e6b3-e73f-4316-9e2b-bf4091ae96bb
```

**Comment by scanum** [comment]
@LanaChin  Changes look correct. Could you please review and merge? Thanks!

**Comment by scanum** [comment]
@officedocsbot assign @LanaChin

**Comment by LanaChin** [comment]
Reviewed changes, added the table back for reference, merged.

**Comment by cadasi** [comment]
Hi  Lana,

I had removed the table because it does not represent the reality anymore, so that could create confusion.... In CSS we received questions from customers about not having that certificate chain and having hybrid issues, but that was not was causing their problem :) the certificate chain is better represented in the link I've added

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**